### ISSUE TITLE
apt-get warning: force-yes is deprecated

### ISSUE THREAD

**Issue by dzedd** [Shippable/support#3526]
I'm not able to install package **libjpeg-dev** with **apt-get**.
I get warning like below:

> W: --force-yes is deprecated, use one of the options starting with --allow instead.

and whole build is marked as failure.

```
before_install:
    - sudo apt-get update
    - sudo apt-get install libjpeg-dev
```

**Comment by rageshkrishna** [comment]
@dzedd Your build isn't failing on that apt-get command. It's failing with `ImportError: cannot import name find_spec` when you are running `python manage.py makemigrations`. [This answer on SO](http://stackoverflow.com/a/34648720/34985) says it's because `find_spec` is only available on Python 3.4+, but your build is currently configured to run on Python 3.3. You should update your YML file to use 3.4 instead.

**Comment by dzedd** [comment]
I changed python version in my yml file and now it works as it should.
